docs/timeseries-reports/sort_contents.md

```diff
@@ -149,7 +149,7 @@ Example:
 
 #### Start points
 
-As seen in [MainComponents](#Main) an activated report that is visible to the user always need one or more starting points.
+As seen in [MainComponents](#Main) an activated report that is visible to the user always needs one or more starting points.
```

Comment:
FYI: @powel-abo, this `[MainComponents](#Main)` anchor is invalid. I'll create an issue for that.

Reply:
https://github.com/Volue-Public/energy-smp-docs/issues/36